src/subscription_manager/managercli.py

```diff
@@ -1043,6 +1041,8 @@ def _do_command(self):
         # Proceed with new registration:
         try:
             if not self.options.activation_keys:
+                print _("Registering to: %s:%s%s") % \
```

Comment:
I like this better, the previous version showing it as a side effect of getting a property was a little odd.